custom/icds/forms.py

```diff
@@ -199,6 +199,8 @@ class CustomSMSReportRequestForm(forms.Form):
     ), required=True)
 
     def __init__(self, *args, **kwargs):
+        disable_submit = kwargs.pop('disable_submit') if 'disable_submit' in kwargs else False
+        print(disable_submit)
```

Comment:
nit: print statement

Reply:
`disable_submit = kwargs.pop('disable_submit', False)` would do.

Reply:
Ohh nice. This is handy.
Thanks 😄